order/views.py

```diff
@@ -1,3 +1,110 @@
-from django.shortcuts import render
+import json
+from datetime import datetime
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from order.models   import Cart, Order
+from user.models      import Address
+
+
+class CartView(View):
+    # @login_required
+    def post(self, request):  # 장바구니에 상품 담기
+        try:
+            data = json.loads(request.body)
+            user_id = "3"  # 데코레이터 나오기 전까지 사용자 임의지정
+
+            # 사용자 장바구니 가져오기, 있으면 가져오고 없으면 생성하여 가져옴
+            if not Order.objects.filter(user_id=user_id, status_id=1).exists():
+                order_cart = Order.objects.create(
+                    user_id           = user_id,
```

Comment:
```suggestion
            if not Order.objects.filter(user_id=user_id, status_id=1).exists():
                new_order = Order.objects.create(
                    user_id           = user_id,
```
Cart는 이 아래 32번째 라인부터 다루기 시작하는 것으로 판단되고, 19번 라인은 주문만을 다루는 것 같은데 변수명에 cart가 포함되어 있으면 다소 헷갈릴 위험이 있을 것 같습니다 :) 따라서 보다 명확하게 주문이라는 것을 알려줄 수 있는 변수명을 추천드립니다. 꼭 `new_order`로 하실 필요는 없고, 의미에 가장 적합하다고 생각되는 단어로 사용해주세요

Reply:
order_cart라는 변수명은 order의 instance인데, status=1(장바구니)인 instance(이 인스턴스를 cart의 instance들이 바라봄)라는 의미에서 사용해 보았습니다.

여기서는 new_order로 정정하도록 하겠습니다.